test/unit/services/domain/test-reduction.js

```diff
@@ -23,78 +23,4 @@ describe('services/domain/reduction', function () {
     expect(reduction.status).to.be.equal(reductionStatusType.ACTIVE)
     expect(reduction.reductionSubmitter).to.equal(150)
   })
-
-  it('should raise a ValidationError if Hours is not supplied', function () {
```

Comment:
all of the actual validate functions are tested closer https://github.com/ministryofjustice/wmt-web/blob/error-list-in-controller/test/unit/services/validators/test-fieldset-validator.js or https://github.com/ministryofjustice/wmt-web/blob/error-list-in-controller/test/unit/services/validators/test-field-validator.js, I didn't think it was worth duplicate tests testing the same underlying function